e2e/ansible/playbooks/resiliency/test-single-replica-failure/test-single-replica-failure-cleanup.yml

```diff
@@ -98,6 +91,3 @@
   delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
   with_items:
     - "{{chaoskube_files}}"
-
```

Comment:
@nsathyaseelan  Why are you using two tasks for removing percona files and chaoskube files?? it can be done in single task right?

Reply:
@swarnakl changed the removing files it will be single task.